app/middleware/cmr-granule-locator.ts

```diff
@@ -124,15 +159,15 @@ async function cmrGranuleLocatorTurbo(
     const queries = sources.map(async (source) => {
       logger.info(`Querying granules for ${source.collection}`, { cmrQuery, collection: source.collection });
       const startTime = new Date().getTime();
-      const maxResults = getMaxGranules(req);
+      const [maxResults, _reason] = getMaxGranules(req, source.collection);
```

Comment:
Unpacking a tuple only to discard it twice in this method and probably elsewhere maybe suggests using an object instead of an array.

Reply:
Yeah, I'm on the fence about that.

Reply:
Done.